### PR TITLE
Patch 1

### PR DESCRIPTION
**Pull request by Fornost461**
Minor fixes (consistency, punctuation, grammar).

### CODE REVIEW

File: source/basics.txt [hunk 1]
```diff
@@ -96,7 +96,7 @@ which is equivalent to
     label.set_halign(Gtk.Align.END)
 
 Instead of using getters and setters you can also get and set the gobject properties
-through the "props" property such as ``widget.props.prop_name = value``.
+through the "props" property such as ``widget.props.prop-name = value``.
```
Comment (sebp) [inline source/basics.txt]: It needs to be "_", a "-" is invalid python.

File: source/unicode.txt [hunk 2]
```diff
@@ -6,7 +6,7 @@ and GTK+ and discusses common errors that arise when working with strings.
 
 Definitions
 -----------
-Conceptional, a string is a list of characters such as
+Conceptually, a string is a list of characters such as
```
Comment (sebp) [inline source/unicode.txt]: There was a previous report (#103) that introduced this change, but I think you are correct in reverting it.

**Comment by sebp** [comment]
I'm closing this issue as no further information has been provided. Please feel free to reopen this issue if you can provide the requested information.

**Comment by Fornost461** [comment]
I think I don't have the rights to reopen the issue. I made a third commit to the branch but it's not appearing here, maybe because the issue is closed.

**Pull request closed without merging**